scai-gen/fileio/pb.go

```diff
@@ -8,12 +8,32 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-func WritePbToFile(pb proto.Message, outFile string) error {
-	pbBytes, err := protojson.Marshal(pb)
+func WritePbToFile(pb proto.Message, outFile string, pretty bool) error {
+	var (
+		pbBytes []byte
+		err     error
+	)
+
+	if pretty {
+		opt := &protojson.MarshalOptions{
```

Comment:
I suggest something like:

```
opt := &protojson.MarshalOptions{}
if pretty {
    opt.Multiline = true
    opt.Indent = "    "
    opt.EmitUnpopulated = false
}

pbBytes, err := opt.Marshal(pb)
if err != nil {
    return err
}
```

You can skip the pbBytes and err declaration above and have a consistent call to Marshal.

Reply:
Yes, that does work a lot better, thank you!